### PR TITLE
add submitBatch script

### PR DESCRIPTION
**Pull request by ChihChengLiang**

### CODE REVIEW

File: scripts/submitBatch.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const contractAddresses = require("../contractAddresses.json");
+
+const RollupContract = require("../build/contracts/Rollup.json");
+
+const argv = require("minimist")(process.argv.slice(2), {
+    string: ["batches"]
+});
+
+/*
+    $ node ./scripts/submitBatch.js --batches batches.json
```
Comment (ChihChengLiang) [inline scripts/submitBatch.js:11]: Need the format of `batches.json`

File: scripts/submitBatch.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        contractAddresses.RollupContract,
+        RollupContract.abi,
+        signer
+    );
+
+    const batches = [];
+
+    for (const [index, batch] of batches.entries()) {
+        txs = "";
+        _updatedRoot = "";
```
Comment (ChihChengLiang) [inline scripts/submitBatch.js:29]: Do we know the stateRoot when producing the batch.json?

File: scripts/submitBatch.js (hunk 1 above)
Comment (ChihChengLiang) [reply scripts/submitBatch.js]: Getting from https://gist.githubusercontent.com/vaibhavchellani/3c1f19925cf2d3a26edf218ff7cff33d/raw/7f4234f7daa6e99043f095df477d93c76708c504/commitments.json

**Comment by ChihChengLiang** [comment]
The script works but if you run it, it has some errors due to #138.

File: scripts/submitBatch.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+/*
+    $ wget https://gist.githubusercontent.com/vaibhavchellani/3c1f19925cf2d3a26edf218ff7cff33d/raw/7f4234f7daa6e99043f095df477d93c76708c504/commitments.json
+    $ node ./scripts/submitBatch.js --batches commitments.json
+*/
+
+async function main() {
+    const batchesFile = argv.batches;
+
+    const provider = new ethers.providers.JsonRpcProvider();
```
Comment (vaibhavchellani) [inline scripts/submitBatch.js:20]: Can you add an cli flag to accept argument, as this accepts only localhost.

**Comment by vaibhavchellani** [comment]
This is perfection.

**Pull request merged**